### ISSUE TITLE
Add FreeBSD release for extended version

### ISSUE THREAD

**Issue by mildred** [gohugoio/hugo#5577]
Some hosting providers are using FreeBSD, and having an extended release of Hugo available for FreeBSD amd64 would be great. Is there anything preventing that?

**Comment by bep** [comment]
>Is there anything preventing that?

A fix for this would make it ... simpler ... https://github.com/goreleaser/goreleaser/issues/705 -- @caarlos0 Christmas wish ...

But even with that; the extended build requires the C tool chain of the target platform installed. Which is why I kind of closed my eyes and said "let's just do this" when I introduced LibSASS and CGO into the build chain. It is still worth it (SASS is really useful), but it makes stuff harder.  Docker makes all of this doable, but I'm waiting for a fix to the above issue before I add more "extended builds" to the mix.

**Comment by caarlos0** [comment]
@bep sorry this is taking too long.. new year's resolution is to do this haha 

https://github.com/goreleaser/goreleaser/issues/705#issuecomment-450738025

**Comment by gfairchild** [comment]
I'd love for an extended FreeBSD release!

**Comment by caarlos0** [comment]
multiple archives is already there, but other parts are still missing...

**Comment by bep** [comment]
@caarlos0 so something like this should work?

```bash
builds:
- binary: hugo
  id: hugo_extended
- binary: hugo
  id: hugo
archives:
  - binaries: ['hugo']
    name_template: 'hugo_{{.Version}}_{{.Os}}-{{.Arch}}'
  - binaries: ['hugo_extended']
    name_template: 'hugo_extended_{{.Version}}_{{.Os}}-{{.Arch}}'
```

**Comment by caarlos0** [comment]
yes, just instead of binaries its `builds` and you use the `build.id`

https://goreleaser.com/customization/#Archive 

still need to finish the releasing parts (which archives to release where), as for now it releases all archives, ex: https://github.com/caarlos0/test/blob/master/.goreleaser.4.yml generates https://github.com/caarlos0/test/releases/tag/v5.6.0

**Comment by HighPriest** [comment]
Any progress on this issue?
I would love an extended hugo binary for a freebsd machine.

**Comment by udanieli** [comment]
You can install extended hugo from [ports](https://www.freshports.org/www/gohugo/), the prebuilt package is fine:

`pkg install gohugo`

If you need it as a dependency, e.g. some specific version or a requirement in some bizarre wrapper (like [this](https://github.com/fenneclab/hugo-bin)), you could need additional workarounds.

**Comment by schrodyn** [comment]
Any progress on this? This issue was opened in 2018... I can't use hugo unto there are FreeBSD releases for the extended version.

**Comment by jmooring** [comment]
I tested this a few days ago. Works great:
<https://gohugo.io/installation/bsd/#freebsd>

**Comment by gfairchild** [comment]
Any update on when this is likely to happen?

**Comment by jmooring** [comment]
Can you do this instead?

sudo pkg install gohugo

**Comment by gfairchild** [comment]
> Can you do this instead?
> 
> sudo pkg install gohugo

Unfortunately not. I use https://www.nearlyfreespeech.net/ for hosting my super simple website, which Hugo is perfect for, and this comes at the cost of my not having root privileges. I just snag the latest release from here, and I've got some scripts that build and deploy everything using my local version of the Hugo binary.
